rollbar/lib/transforms/scruburl.py

```diff
@@ -24,12 +24,7 @@ def __init__(self,
         self.params_to_scrub = set(map(lambda x: x.lower(), params_to_scrub))
 
     def in_scrub_fields(self, key):
-        if not key:
-            # This can happen if the transform is applied to a non-object,
-            # like a string.
-            return True
-
-        return super(ScrubUrlTransform, self).in_scrub_fields(key)
+        return True
```

Comment:
This is a little unintuitive and sorta looks like it could be debug code - can you add a comment explaining why we always return `True` here?